mcs/class/corlib/System/Array.cs

```diff
@@ -624,9 +599,40 @@ public static void ConstrainedCopy (Array sourceArray, int sourceIndex, Array de
 			Copy (sourceArray, sourceIndex, destinationArray, destinationIndex, length);
 		}
 
-		object GetValueWithFlattenedIndex_NoErrorCheck (int flattenedIndex)
+		public static T[] Empty<T>()
+		{
+			return EmptyArray<T>.Value;
+		}
+
+		public void Initialize()
+		{
```

Comment:
Yes but we never supported it.

https://msdn.microsoft.com/en-us/library/system.array.initialize(v=vs.110).aspx